### PR TITLE
Add prevent-serp-tracking package

### PR DESCRIPTION
**Pull request by smalluban**

### CODE REVIEW

**Review by philipp-classen — APPROVED** [review]

File: packages/prevent-serp-tracking/src/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * WhoTracks.Me
+ * https://whotracks.me/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+ function safeLinkClick(event) {
```
Comment (chrmod) [inline packages/prevent-serp-tracking/src/index.js:11]: please remove the white space

File: packages/prevent-serp-tracking/src/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  const targetUrl =
+    el.pathname === '/url' && new URL(el.href).searchParams.get('url');
+
+  if (targetUrl) {
+    event.stopImmediatePropagation();
+    el.href = targetUrl;
+  }
+}
+
+document.addEventListener('click', safeLinkClick, true);
```
Comment (chrmod) [inline packages/prevent-serp-tracking/src/index.js:28]: what about touch events?

**Review by chrmod — CHANGES_REQUESTED** [review]

File: packages/prevent-serp-tracking/src/index.js (hunk 2 above)
Comment (philipp-classen) [reply packages/prevent-serp-tracking/src/index.js:28]: I think, it was not necessary. But you can double-check on an iPhone/iPad.

Comment (smalluban) [reply packages/prevent-serp-tracking/src/index.js:28]: The click event is the last one, which is always triggered, on the touch devices also, just after the `touchend`, mainly for the backwards compatibility. 

I can check on iOS, however I would need to add my ipad to our provisional profile, as now I can't build it for iOS.

**Review by chrmod — APPROVED** [review]

**Pull request merged**